python/apps/taiga/src/taiga/workspaces/workspaces/repositories.py

```diff
@@ -294,3 +358,13 @@ def delete_workspaces(filters: WorkspaceFilters = {}) -> int:
     qs = _apply_filters_to_queryset(qs=DEFAULT_QUERYSET, filters=filters)
     count, _ = qs.delete()
     return count
+
+
+##########################################################
+# misc
+##########################################################
+
+
+@sync_to_async
```

Comment:
If it's returning Projects, wouldn't it be better in the project's repositories?

Reply:
In the other way, it's simply accessing an attribute, and not performing a Query. I'm not sure what criteria to follow, both could be valid I guess..

Reply:
Is in ws because is only for projects in ws, to access a projects.all() because I need it.

Reply:
it's ok for me.